Tests/SnapshotConfiguration.swift

```diff
@@ -79,6 +83,16 @@ extension SnapshotConfiguration {
     "LottieFiles/gradient_shapes": .testWithExperimentalRenderingEngine,
     "TypeFace/H": .testWithExperimentalRenderingEngine,
     "TypeFace/O": .testWithExperimentalRenderingEngine,
+
+    /// Test cases for the `AnimationKeypath` / `AnyValueProvider` system
+    "Switch": .init(
```

Comment:
One reason I lean a bit towards just customizing the existing snapshot, instead of adding a new separate `Switch_CustomColors.json` (as an example), is that the new file would show up in the Example app alongside `Switch.json` but wouldn't have any custom colors there (since the Example app doesn't include this custom per-snapshot configuration).

Reply:
This most scalable option for this long-term may be to automatically create two test cases for snapshots with `customValueProviders` -- one snapshot with the value providers applied, and one without them applied.

Reply:
> This most scalable option for this long-term may be to automatically create two test cases for snapshots with `customValueProviders` -- one snapshot with the value providers applied, and one without them applied.

This seems like a reasonable solve in a a follow up!